File: julia/graphs/graph1.jl
# Open the terminal: control + shift + enter
# Open the command palette: control + shift + p
# Run Julia: shift + enter

# Install packages (type in the Julia environment)
    # ] add Statistics
    # ] add Distributions
    # ] add LinearAlgebra
    # ] add Plots
    # ] add DataFrames
    # ] add SymPy

# Load useful packages
#using Statistics
#using Distributions
#using LinearAlgebra
#using Plots 
#using DataFrames
#using SymPy

# Shorcuts
    # Compile julia in bash of Git: julia mi_script.jl
    # Ctrl + s: save change

# Example: Create a simple plot in Julia using the Plots library
# Import the Plots library
using Plots
# Generate some data for the plot
x = 1:10
y = rand(10)
# Create a line plot
plot(x, y, label="Random Data", xlabel="X-axis", 
     ylabel="Y-axis", title="Simple Plot")
# Save figure
savefig("figures/graph1.png")
savefig("figures/graph1.pdf")

File: python/graphs/graph1.py
# Shorcuts 
    # ctrl + s: save
    # ctrl + ñ: open terminal
    # pip install matplotlib

# Library import
import matplotlib.pyplot as plt
import numpy as np
# Data for the line plot
x_line = np.linspace(0, 10, 100)
y_line = np.sin(x_line)
# Data for the bar plot
categories = ['Category A', 'Category B',
               'Category C', 'Category D']
values = [25, 50, 75, 100]
# Creating the line plot
plt.figure(figsize=(10, 4))
plt.subplot(1, 2, 1)  # 1 row, 2 columns, this is the first plot
plt.plot(x_line, y_line, label='sin(x)',
          color='blue', marker='o')
plt.title('Line Plot')
plt.xlabel('X-axis')
plt.ylabel('Y-axis')
plt.legend()
# Creating the bar plot
plt.subplot(1, 2, 2)  # 1 row, 2 columns, this is the second plot
plt.bar(categories, values, color='green')
plt.title('Bar Plot')
plt.xlabel('Categories')
plt.ylabel('Values')
# Adjust layout to prevent overlapping
plt.tight_layout()
# Save figure
plt.savefig("figures/graph1.png")
# Show the plots
plt.show()

